mints/06.json

```diff
@@ -0,0 +1 @@
+{"name":"skill/valory/decision_maker_abci:0.1.0","description":"ABCI application for common apps.","code_uri":"ipfs://bafybeigircx36tx3nw3r6dgym357e3wyzabb7y7zhahhttwbkgnxinsgre","image":"ipfs://bafybeiasruosr2t2qzv4iqoaj3l5idztygk5q36khovdfrdwzbddwmi3km","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```

Comment:
This description of the skill is not a good one

Reply:
Improved on https://github.com/valory-xyz/trader/pull/63.